src/deye_events.py

```diff
@@ -46,6 +64,18 @@ class DeyeLoggerStatusEvent(DeyeEvent):
     def __init__(self, online: bool):
         self.online = online
 
+    def __str__(self) -> str:
+        return "online" if self.online else "offline"
+
+    def __eq__(self, other) -> bool:
+        return self.online == getattr(other, "online", None)
```

Comment:
Why do you use `getattr` here? This looks like duck-typing to me? Why not to check the instance type of `other` in the first place?

Reply:
thought it may cause issues if compared event type is not same, but seems to be unnecessary
simplified now